src/components/federated-solr-faceted-search.js

```diff
@@ -98,14 +98,25 @@ class FederatedSolrFacetedSearch extends React.Component {
       pageTitle = <h1>{this.props.options.pageTitle}</h1>;
     }
 
+    // Grab env vars.
+    const {
+      containerClass = '',
```

Comment:
Did we lose some default values here?

Reply:
The optional custom classes would be additional to existing classes if needed so the default is to have none.